### PR TITLE
Lock oc_erchef to 1.0 while keeping jiffy at latest

### PR DESCRIPTION
**Pull request by stevendanna**
chef-mover and moser depend on modules provided by oc_erchef.  To ensure that the upgrades work correctly into the future, the version of oc_erchef we depend on must be locked.  Otherwise, changes to the modules in oc_erchef could break the migration scripts.  

This is NOT the long term solution, but will allow us to continue to safely put out new builds without breaking upgrades.

Jiffy has been manually bumped to a recent revision that works on RHEL 7

### CODE REVIEW

**Comment by stevendanna** [comment]
cc @chef/lob

**Comment by tylercloke** [comment]
:+1:

File: src/mover_reindex_migration_callback.erl [hunk 1]
```diff
@@ -6,11 +6,15 @@
...
          error_halts_migration/0,
-         reconfigure_object/1
+         reconfigure_object/1,
+         needs_account_dets/0
          ]).
 
 -include_lib("moser/include/moser.hrl").
 
+needs_account_dets() ->
+    true.
```
Comment (tylercloke) [inline src/mover_reindex_migration_callback.erl]: Yay for changing interfaces :(

**Comment by danielsdeleo** [comment]
LGTM

**Pull request merged**